platform/linux-generic/test/example/classifier/pktio_env

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+#
+# Copyright (c) 2020, Marvell
+# All rights reserved.
+#
+# SPDX-License-Identifier:	BSD-3-Clause
+#
+# Script to setup interfaces used for running application on linux-generic.
+#
+# For linux-generic the default behavior is to create one pcap interfaces
+# and uses udp64.pcap to inject traffic.
+#
+# Network set-up
+# +---------+          +-----------+
+# |pcap intf| IF0<---> | Classifier|
+# +---------           +-----------+
+#
+
+PCAP_IN=`find . ${TEST_DIR} $(dirname $0) -name udp64.pcap -print -quit`
+echo "using PCAP in=${PCAP_IN}"
+
+IF0=pcap:in=${PCAP_IN}
+TIME_OUT_VAL=60
+ret=0
+
+if [ "$0" = "$BASH_SOURCE" ]; then
+	echo "Error: Platform specific env file has to be sourced."
+fi
+
+validate_result()
+{
+	echo "Validate results"
+	return 0;
+}
+
+setup_interfaces()
+{
+	echo "pktio: setting up test interfaces $IF0"
```

Comment:
We have a single test interface. This print doesn't provide much value, so it could be removed.

Reply:
Ack